lock-api/src/main/java/com/palantir/lock/LockResponse.java

```diff
@@ -103,10 +114,19 @@ public boolean success() {
         return token;
     }
 
+    @JsonIgnore
     @Nullable public LockRefreshToken getLockRefreshToken() {
         return token == null ? null : token.getLockRefreshToken();
     }
 
+    public boolean isBlockAndRelease() {
+        return isBlockAndRelease;
+    }
+
+    public List<LockWithClient> getLocks() {
+        return ImmutableList.copyOf(Iterables.transform(lockHolders.entrySet(), TO_LOCK_WITH_CLIENT_FUNCTION));
```

Comment:
nit: the function could probably just be inline here

Reply:
Done, replaced with stream.